[2022-10-05T00:30:13.693+0000] {processor.py:156} INFO - Started process (PID=49) to work on /opt/airflow/dags/my_dags.py
[2022-10-05T00:30:13.724+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dags.py for tasks to queue
[2022-10-05T00:30:13.725+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:30:13.725+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dags.py
[2022-10-05T00:30:17.586+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dags.py
[2022-10-05T00:30:17.638+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dags.py took 3.964 seconds
[2022-10-05T00:31:43.867+0000] {processor.py:156} INFO - Started process (PID=130) to work on /opt/airflow/dags/my_dags.py
[2022-10-05T00:31:43.868+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dags.py for tasks to queue
[2022-10-05T00:31:43.869+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:31:43.869+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dags.py
[2022-10-05T00:31:43.995+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:31:43.918+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dags.py", line 3, in <module>
    from airflow.operators.python.PythonOperator import PythonOperator
ModuleNotFoundError: No module named 'airflow.operators.python.PythonOperator'; 'airflow.operators.python' is not a package
[2022-10-05T00:31:43.998+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dags.py
[2022-10-05T00:31:44.159+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dags.py took 0.326 seconds
[2022-10-05T00:32:34.076+0000] {processor.py:156} INFO - Started process (PID=40) to work on /opt/airflow/dags/my_dags.py
[2022-10-05T00:32:34.080+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dags.py for tasks to queue
[2022-10-05T00:32:34.086+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:32:34.085+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dags.py
[2022-10-05T00:32:34.147+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:32:34.132+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dags.py", line 3, in <module>
    from airflow.operators.python.PythonOperator import PythonOperator
ModuleNotFoundError: No module named 'airflow.operators.python.PythonOperator'; 'airflow.operators.python' is not a package
[2022-10-05T00:32:34.151+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dags.py
[2022-10-05T00:32:34.302+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dags.py took 0.234 seconds
[2022-10-05T00:34:23.547+0000] {processor.py:156} INFO - Started process (PID=39) to work on /opt/airflow/dags/my_dags.py
[2022-10-05T00:34:23.554+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dags.py for tasks to queue
[2022-10-05T00:34:23.555+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:34:23.555+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dags.py
[2022-10-05T00:34:23.609+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:34:23.593+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dags.py", line 3, in <module>
    from airflow.operators.python.PythonOperator import PythonOperator
ModuleNotFoundError: No module named 'airflow.operators.python.PythonOperator'; 'airflow.operators.python' is not a package
[2022-10-05T00:34:23.611+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dags.py
[2022-10-05T00:34:23.721+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dags.py took 0.188 seconds
[2022-10-05T00:36:34.633+0000] {processor.py:156} INFO - Started process (PID=48) to work on /opt/airflow/dags/my_dags.py
[2022-10-05T00:36:34.637+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dags.py for tasks to queue
[2022-10-05T00:36:34.641+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:36:34.641+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dags.py
[2022-10-05T00:36:34.733+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:36:34.705+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dags.py", line 3, in <module>
    from airflow.operators.python.PythonOperator import PythonOperator
ModuleNotFoundError: No module named 'airflow.operators.python.PythonOperator'; 'airflow.operators.python' is not a package
[2022-10-05T00:36:34.735+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dags.py
[2022-10-05T00:36:34.905+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dags.py took 0.312 seconds
[2022-10-05T00:37:55.489+0000] {processor.py:156} INFO - Started process (PID=113) to work on /opt/airflow/dags/my_dags.py
[2022-10-05T00:37:55.508+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dags.py for tasks to queue
[2022-10-05T00:37:55.516+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:37:55.516+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dags.py
[2022-10-05T00:37:55.642+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:37:55.595+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dags.py", line 3, in <module>
    from airflow.operators.python.PythonOperator import PythonOperator
ModuleNotFoundError: No module named 'airflow.operators.python.PythonOperator'; 'airflow.operators.python' is not a package
[2022-10-05T00:37:55.643+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dags.py
[2022-10-05T00:37:55.989+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dags.py took 0.543 seconds
[2022-10-05T00:38:39.368+0000] {processor.py:156} INFO - Started process (PID=39) to work on /opt/airflow/dags/my_dags.py
[2022-10-05T00:38:39.373+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dags.py for tasks to queue
[2022-10-05T00:38:39.377+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:38:39.377+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dags.py
[2022-10-05T00:38:39.432+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:38:39.415+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dags.py", line 3, in <module>
    from airflow.operators.python.PythonOperator import PythonOperator
ModuleNotFoundError: No module named 'airflow.operators.python.PythonOperator'; 'airflow.operators.python' is not a package
[2022-10-05T00:38:39.433+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dags.py
[2022-10-05T00:38:39.564+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dags.py took 0.203 seconds
[2022-10-05T00:40:03.328+0000] {processor.py:156} INFO - Started process (PID=116) to work on /opt/airflow/dags/my_dags.py
[2022-10-05T00:40:03.338+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dags.py for tasks to queue
[2022-10-05T00:40:03.340+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:40:03.340+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dags.py
[2022-10-05T00:40:03.430+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:40:03.420+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dags.py", line 3, in <module>
    from airflow.operators.python.PythonOperator import PythonOperator
ModuleNotFoundError: No module named 'airflow.operators.python.PythonOperator'; 'airflow.operators.python' is not a package
[2022-10-05T00:40:03.436+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dags.py
[2022-10-05T00:40:03.647+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dags.py took 0.329 seconds
[2022-10-05T00:40:54.622+0000] {processor.py:156} INFO - Started process (PID=47) to work on /opt/airflow/dags/my_dags.py
[2022-10-05T00:40:54.625+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dags.py for tasks to queue
[2022-10-05T00:40:54.636+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:40:54.636+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dags.py
[2022-10-05T00:40:54.725+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:40:54.700+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dags.py", line 3, in <module>
    from airflow.operators.python.PythonOperator import PythonOperator
ModuleNotFoundError: No module named 'airflow.operators.python.PythonOperator'; 'airflow.operators.python' is not a package
[2022-10-05T00:40:54.727+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dags.py
[2022-10-05T00:40:54.931+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dags.py took 0.357 seconds
[2022-10-05T00:42:19.274+0000] {processor.py:156} INFO - Started process (PID=114) to work on /opt/airflow/dags/my_dags.py
[2022-10-05T00:42:19.283+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dags.py for tasks to queue
[2022-10-05T00:42:19.296+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:42:19.295+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dags.py
[2022-10-05T00:42:19.364+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:42:19.352+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dags.py", line 3, in <module>
    from airflow.operators.python.PythonOperator import PythonOperator
ModuleNotFoundError: No module named 'airflow.operators.python.PythonOperator'; 'airflow.operators.python' is not a package
[2022-10-05T00:42:19.371+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dags.py
[2022-10-05T00:42:19.604+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dags.py took 0.357 seconds
[2022-10-05T00:43:02.203+0000] {processor.py:156} INFO - Started process (PID=41) to work on /opt/airflow/dags/my_dags.py
[2022-10-05T00:43:02.207+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dags.py for tasks to queue
[2022-10-05T00:43:02.210+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:43:02.210+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dags.py
[2022-10-05T00:43:02.295+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:43:02.278+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dags.py", line 3, in <module>
    from airflow.operators.python.PythonOperator import PythonOperator
ModuleNotFoundError: No module named 'airflow.operators.python.PythonOperator'; 'airflow.operators.python' is not a package
[2022-10-05T00:43:02.298+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dags.py
[2022-10-05T00:43:02.430+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dags.py took 0.264 seconds
[2022-10-05T00:45:44.529+0000] {processor.py:156} INFO - Started process (PID=46) to work on /opt/airflow/dags/my_dags.py
[2022-10-05T00:45:44.567+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dags.py for tasks to queue
[2022-10-05T00:45:44.594+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:45:44.585+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dags.py
[2022-10-05T00:45:48.311+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dags.py
[2022-10-05T00:45:56.899+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dags.py took 12.441 seconds
[2022-10-05T00:48:37.397+0000] {processor.py:156} INFO - Started process (PID=39) to work on /opt/airflow/dags/my_dags.py
[2022-10-05T00:48:37.400+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dags.py for tasks to queue
[2022-10-05T00:48:37.402+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:48:37.401+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dags.py
[2022-10-05T00:48:37.413+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:48:37.410+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dags.py", line 34
    s3Update = PythonOperator(
           ^
IndentationError: expected an indented block
[2022-10-05T00:48:37.415+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dags.py
[2022-10-05T00:48:46.493+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dags.py took 9.106 seconds
[2022-10-05T00:51:05.524+0000] {processor.py:156} INFO - Started process (PID=39) to work on /opt/airflow/dags/my_dags.py
[2022-10-05T00:51:05.525+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dags.py for tasks to queue
[2022-10-05T00:51:05.528+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:51:05.527+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dags.py
[2022-10-05T00:51:08.508+0000] {processor.py:768} INFO - DAG(s) dict_keys(['dag']) retrieved from /opt/airflow/dags/my_dags.py
[2022-10-05T00:52:29.365+0000] {processor.py:156} INFO - Started process (PID=112) to work on /opt/airflow/dags/my_dags.py
[2022-10-05T00:52:29.366+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dags.py for tasks to queue
[2022-10-05T00:52:29.367+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:52:29.367+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dags.py
[2022-10-05T00:53:29.934+0000] {processor.py:156} INFO - Started process (PID=40) to work on /opt/airflow/dags/my_dags.py
[2022-10-05T00:53:29.936+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dags.py for tasks to queue
[2022-10-05T00:53:29.944+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:53:29.944+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dags.py
[2022-10-05T00:53:33.848+0000] {processor.py:768} INFO - DAG(s) dict_keys(['dag']) retrieved from /opt/airflow/dags/my_dags.py
[2022-10-05T00:53:33.936+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:53:33.936+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-05T00:53:33.957+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:53:33.957+0000] {dag.py:2594} INFO - Creating ORM DAG for dag
[2022-10-05T00:53:33.972+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:53:33.972+0000] {dag.py:3328} INFO - Setting next_dagrun for dag to 2022-10-04T00:00:00+00:00, run_after=2022-10-05T00:00:00+00:00
[2022-10-05T00:54:02.276+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dags.py took 32.354 seconds
[2022-10-05T00:55:01.275+0000] {processor.py:156} INFO - Started process (PID=96) to work on /opt/airflow/dags/my_dags.py
[2022-10-05T00:55:01.284+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dags.py for tasks to queue
[2022-10-05T00:55:01.285+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:55:01.285+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dags.py
[2022-10-05T00:55:40.439+0000] {processor.py:156} INFO - Started process (PID=39) to work on /opt/airflow/dags/my_dags.py
[2022-10-05T00:55:40.447+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dags.py for tasks to queue
[2022-10-05T00:55:40.448+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:55:40.448+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dags.py
[2022-10-05T00:55:43.367+0000] {processor.py:768} INFO - DAG(s) dict_keys(['dag']) retrieved from /opt/airflow/dags/my_dags.py
[2022-10-05T00:55:43.577+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:55:43.572+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-05T00:55:43.705+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:55:43.704+0000] {dag.py:3328} INFO - Setting next_dagrun for dag to 2022-10-04T00:00:00+00:00, run_after=2022-10-05T00:00:00+00:00
[2022-10-05T00:55:55.182+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dags.py took 14.762 seconds
[2022-10-05T00:57:46.113+0000] {processor.py:156} INFO - Started process (PID=40) to work on /opt/airflow/dags/my_dags.py
[2022-10-05T00:57:46.118+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dags.py for tasks to queue
[2022-10-05T00:57:46.127+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:57:46.127+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dags.py
[2022-10-05T00:57:49.192+0000] {processor.py:768} INFO - DAG(s) dict_keys(['dag']) retrieved from /opt/airflow/dags/my_dags.py
[2022-10-05T00:57:49.304+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:57:49.304+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-05T00:57:49.346+0000] {logging_mixin.py:117} INFO - [2022-10-05T00:57:49.345+0000] {dag.py:3328} INFO - Setting next_dagrun for dag to 2022-10-04T00:00:00+00:00, run_after=2022-10-05T00:00:00+00:00
